manifests/deployment-config.yaml

```diff
@@ -0,0 +1,43 @@
+---
+kind: DeploymentConfig
+apiVersion: apps.openshift.io/v1
+metadata:
+  name: peribolos-as-services
+spec:
+  test: false
```

Comment:
What does `test` mean in here?

Reply:
It was property in dc https://docs.openshift.com/container-platform/3.11/rest_api/apps_openshift_io/deploymentconfig-apps-openshift-io-v1.html#.spec

seems like it is not there anymore in 4.x ocp
will remove it in the next enhancement pr.